### PR TITLE
Helm charts

### PR DESCRIPTION
**Pull request by Pipazoul**
Helm chart template to deploy the datatlas stack on a kubernetes cluster in one command line
Does not handle db dumps init yet

### CODE REVIEW

**Comment by Pipazoul** [comment]
@lutangar  I'll need some help for the linter to exclude the helm charts files :)

File: helm/example.values.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+backend:
```
Comment (lutangar) [inline helm/example.values.yml]: Don't know what's the convention here, but this Yaml file is the only one with the `yml`  extension while the others have the `yaml` extension. But both probably work !

**Comment by lutangar** [comment]
> @lutangar I'll need some help for the linter to exclude the helm charts files :)

Alright, let's see if 1edff34 is fixing the pipeline.

File: helm/example.values.yml (hunk 1 above)
Comment (Pipazoul) [reply helm/example.values.yml]: You're right, I've renamed the file

**Pull request merged**